src/bend.ts

```diff
@@ -101,11 +99,10 @@ export class Bend extends Modifier {
     this.xShift = 0;
     this.tap = '';
     this.renderOptions = {
-      lineWidth: 1.5,
-      lineStyle: '#777777',
       bendWidth: 8,
       releaseWidth: 8,
     };
+    this.setStyle({ lineWidth: 1.0, strokeStyle: 'black', fillStyle: 'black' });
```

Comment:
Should this be customizable? I think the goal is to have all the rendering options in Metrics, and later allow developers to customize everything, right?

Reply:
This is customizable calling SetSyle on the element. This is just the default set in the constructor.

There is a mix in the code where styles are in renderOptions. After Bend, I will be fixing the other ones.